usr/share/rear/conf/default.conf

```diff
@@ -3376,15 +3376,34 @@ ELILO_BIN=
 
 ################ ---- custom scripts
 #
-# NOTE: The scripts can be defined as an array to better handly spaces in parameters.
+# NOTE: The scripts can be defined as an array to better handle spaces in parameters.
 # The scripts are called like this: eval "${PRE_RECOVERY_SCRIPT[@]}"
+#
+# Alternatively, commands can be executed by using the corresponding
+# XXX_COMMANDS array variable, which evaluates as shown below:
+#
+#   for command in "${XXX_COMMANDS[@]}"; do
+#     eval "$command"
+#   done
+#
+# Using XXX_COMMANDS is preferred because it's less error-prone when requiring
+# multiple commands to be executed.
+# For example,
+#   PRE_RECOVERY_SCRIPT=( 'multipathd' ';' 'sleep 3' )
+# can be rewritten as
+#   PRE_RECOVERY_COMMANDS=( 'multipathd', 'sleep 3' )
```

Comment:
No comma between array elements - so it should be
`PRE_RECOVERY_COMMANDS=( 'multipathd', 'sleep 3' )`

Reply:
@jsmeix you mean `PRE_RECOVERY_COMMANDS=( 'multipathd' 'sleep 3' )`, I suppose